### PR TITLE
Predicate save bonuses against spells on the item type

### PR DESCRIPTION
**Pull request by VestOfHolding**

### CODE REVIEW

**Comment by VestOfHolding** [comment]
Looks like I picked up a recent change to ranges in running extractPacks on the feats.

**Comment by stwlam** [comment]
> Looks like I picked up a recent change to ranges in running extractPacks on the feats.

You're reverting them.

**Comment by VestOfHolding** [comment]
> > Looks like I picked up a recent change to ranges in running extractPacks on the feats.
> 
> You're reverting them.

Oh, well that's definitely no good. I'll remove those files from this change then.

File: packs/data/equipment.db/staff-of-the-magi.json [hunk 1]
```diff
@@ -86,10 +86,9 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Staff of the Magi (vs. spells)",
                 "predicate": {
-                    "all": [
-                        "spells"
+                    "any": [
```
Comment (SpartanCPA) [inline packs/data/equipment.db/staff-of-the-magi.json:90]: Why change to any? There is only one option. It is still effectively an all...

File: packs/data/feats.db/bloodline-resistance.json [hunk 2]
```diff
@@ -29,10 +29,7 @@
                 "predicate": {
                     "any": [
                         "magical",
-                        "arcane",
-                        "divine",
-                        "occult",
-                        "primal"
```
Comment (SpartanCPA) [inline packs/data/feats.db/bloodline-resistance.json:35]: You want to keep these. Many effects have one of the tradition traits, but not the magical traits. All effects with a tradition trait inherit the magical trait by default, and would still trigger the bonus.

File: packs/data/equipment.db/staff-of-the-magi.json (hunk 1 above)
Comment (VestOfHolding) [reply packs/data/equipment.db/staff-of-the-magi.json:90]: Oh, just copy paste. Nothing specific.

File: packs/data/feats.db/bloodline-resistance.json [hunk 3]
```diff
@@ -29,10 +29,7 @@
                 "predicate": {
                     "any": [
                         "magical",
```
Comment (SpartanCPA) [inline packs/data/feats.db/bloodline-resistance.json:31]:
```suggestion
                        "magical",
                        "arcane",
                        "divine",
                        "occult",
                        "primal",
```

File: packs/data/feats.db/pervasive-superstition.json [hunk 4]
```diff
@@ -32,6 +32,18 @@
...
                 "key": "RollOption",
                 "option": "orc-superstition"
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": {
+                    "any": [
+                        "magical",
+                        "item:type:spell"
+                    ]
```
Comment (SpartanCPA) [inline packs/data/feats.db/pervasive-superstition.json:42]:
```suggestion
                    "any": [
                        "magical",
                        "arcane",
                        "divine",
                        "occult",
                        "primal",
                        "item:type:spell"
                    ]
```

File: packs/data/equipment.db/staff-of-the-magi.json (hunk 1 above)
Comment (SpartanCPA) [reply packs/data/equipment.db/staff-of-the-magi.json:90]:
```suggestion
                    "all": [
```

File: packs/data/equipment-effects.db/effect-spellguard-shield.json [hunk 5]
```diff
@@ -19,7 +20,7 @@
                 "label": "Spellguard Shield (vs. spells)",
```
Comment (SpartanCPA) [inline packs/data/equipment-effects.db/effect-spellguard-shield.json:20]: Please add source

**Review by SpartanCPA — CHANGES_REQUESTED** [review]

File: packs/data/equipment-effects.db/effect-spellguard-shield.json (hunk 5 above)
Comment (VestOfHolding) [reply packs/data/equipment-effects.db/effect-spellguard-shield.json:20]: Done.

**Comment by stwlam** [comment]
Fix PR title :)

File: packs/data/backgrounds.db/tech-reliant.json [hunk 6]
```diff
@@ -36,6 +36,10 @@
                 "key": "FlatModifier",
                 "predicate": {
                     "any": [
+                        "arcane",
```
Comment (SpartanCPA) [inline packs/data/backgrounds.db/tech-reliant.json:39]:
```suggestion
                        "magical",
                        "arcane",
```

**Review by SpartanCPA — APPROVED** [review]

**Pull request merged**